rename.go

```diff
@@ -12,7 +12,7 @@ import (
 func (c *Client) Rename(oldpath, newpath string) error {
 	_, err := c.getFileInfo(newpath)
 	if err == nil {
-		return &os.PathError{"rename", newpath, os.ErrExist}
+		// Do nothing, the existing newpath will be overwritten
```

Comment:
I kept this `getFileInfo`, as we need this to verify if we have permission to write to newpath.